toonz/sources/toonzqt/doublepairfield.cpp

```diff
@@ -98,19 +98,11 @@ void DoubleValuePairField::paintEvent(QPaintEvent *) {
   int x1 = value2pos(m_maxValue);
   int y  = height() / 2;
 
-  p.setPen(getLightLineColor());
-  p.drawLine(x0 - 1, y, x1, y);
-  p.setPen(getLightLineEdgeColor());
-  p.drawPoint(x1 + 1, y);
-  p.drawPoint(x1 + 1, y - 1);
-  p.drawPoint(x1 + 1, y - 2);
-  p.drawPoint(x1 + 1, y - 3);
-  p.setPen(getMiddleLineColor());
-  p.drawLine(x0, y - 2, x1 - 1, y - 2);
   p.setPen(getDarkLineColor());
   p.drawLine(x0 - 1, y - 3, x1, y - 3);
-  p.drawPoint(x0 - 1, y - 1);
-  p.drawPoint(x0 - 1, y - 2);
+  p.drawLine(x0 - 1, y - 2, x1, y - 2);
+  p.drawLine(x0 - 1, y - 1, x1, y - 1);
+  p.drawLine(x0 - 1, y, x1, y);
```

Comment:
How about making it simple, drawing 4-pixel-width line instead of 4 lines?
Lines 101-105 can be replaced by;
```
p.setPen(QPen(getDarkLineColor(),4));
p.drawLine(x0 - 1, y, x1, y);
```